drivers/SPISlave.cpp

```diff
@@ -45,17 +45,17 @@ void SPISlave::frequency(int hz)
 
 int SPISlave::receive(void)
 {
-    return (spi_slave_receive(&_spi));
+    return 0;//(spi_slave_receive(&_spi));
```

Comment:
Why is function commented out? Was the intent to remove it?

Reply:
It looks like I forgot the commit SPISlave driver changes. This should be ok now.